examples/tensorflow/classification/main.py

```diff
@@ -256,8 +255,6 @@ def run(config):
                                               callbacks=callbacks,
                                               initial_epoch=initial_epoch,
                                               steps_per_epoch=train_steps,
-                                              tensorboard_writer=SummaryWriter(config.log_dir,
```

Comment:
Runners require a raw tensorboard writer because it calls `.add_scalar()` and `.add_image()` methods. This `SummaryWriter` is our custom summary writer and it does not implement these methods

Reply:
Thanks for you reply. Will TensorFlow classification example dump statistics in the tensorboard after these changes?

Reply:
You are right, it won't, fixed this